File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--read-from-and-write-to-document-properties_2.vb
        If prop.Value Is Nothing Then
            prop.Value = 1
        Else
            Dim revision As Integer
            If Integer.TryParse(prop.Value.ToString(), revision) Then
                prop.Value = revision + 1
                MessageBox.Show("Revision Number = " & revision)
            Else
                MessageBox.Show("Revision Number = invalid value")
            End If
        End If

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--read-from-and-write-to-document-properties_2.cs
            if (prop.Value == null)
            {
                prop.Value = 1;
            }
            else
            {
                int revision;
                if (int.TryParse((string)prop.Value, out revision))
                {
                    prop.Value = revision + 1;
                    MessageBox.Show("Revision Number = " + revision);
                }
                else
                {
                    MessageBox.Show("Revision Number = invalid value");
                }
            }